content/posts/react-query-leveraging-query-context/index.mdx

```diff
@@ -245,7 +245,7 @@ const fetchTodos = async ({
 export const useTodos = () => {
   const { state, sorting } = useTodoParams()
 
-  return useQuery(todoKeys.list(state, sorting), fetchTodos)
+  return useQuery(todoKeys.list(state, sorting), () => fetchTodos)
```

Comment:
no, that's not needed and actually doesn't work for this example :)

if we want an inline function, we would need to pass the `QueryFunctionContext` injected by react-query as well:

```
return useQuery(todoKeys.list(state, sorting), (context) => fetchTodos(context))
```

the point free notation, passing the function as a reference, does the same.